From 46d85372bb25f124550b9734c9ea7bc971529b9d Mon Sep 17 00:00:00 2001
From: jinghua <xigu@marvell.com>
Date: Thu, 23 Mar 2017 19:28:25 +0800
Subject: [PATCH 0917/1345] dts: a3700: add dt node for watchdog driver

commit  703e3f593651660bb05dc163d02698160f87e774 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

This patch adds watchdog DT node for Armada 3700, which
by default is enabled.

Change-Id: I336a319d162bdb02593706e7fd933f55dd0d82f9
Signed-off-by: jinghua <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37896
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-by: Hua Jing <jinghua@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 18f74b5..976ccc2 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -316,6 +316,13 @@
 				#clock-cells = <1>;
 			};
 
+			wdt: watchdog-timer@8300 {
+				compatible = "marvell,armada-3700-wdt";
+				reg = <0xd064 0x4>,
+				      <0x8300 0x40>;
+				clocks = <&osc>;
+			};
+
 			npinctrl: north-bridge-pinctrl@13830 {
 				compatible = "marvell,armada-3700-nb-pinctrl";
 				reg = <0x13830 0x4>;
-- 
1.7.9.5

